applications/app/views/fragments/audioBody.scala.html

```diff
@@ -9,6 +9,8 @@
 
 @trackingCode(target: String) = {@if(page.media.tags.isPodcast){podcast:subscribe:}@target:@page.media.metadata.webTitle}
 
+@seriesId() = {@audio.tags.tags.find(_.properties.podcast.nonEmpty).map(s => s.id.substring(s.id.lastIndexOf("/") + 1))}
```

Comment:
`.trim`?

Reply:
this is taking `news/series/todayinfocus` into `todayinfocus`, I don't think `trim` will help?